src/sardana/tango/core/util.py

```diff
@@ -932,7 +932,12 @@ def get_logstash_conf(dev_name):
             port = int(props["LogstashPort"][0])
         except IndexError:
             port = 12345
```

Comment:
It is not part of this PR, but I just found it. I think this is not correct. I think it is beter to move write it as the default property value in its declaration and documente it accordingly.

Reply:
OK, I'll update this.